docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+services:
+  myservice:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    image: result/latest
```

Comment:
Won't this produce an image named `result/latest` once you start this, is that necessary?